projects/plugins/protect/jetpack-protect.php

```diff
@@ -97,7 +97,10 @@ function () {
  * @param string $plugin Path to the plugin file relative to the plugins directory.
  */
 function jetpack_protect_plugin_activation( $plugin ) {
-	if ( JETPACK_PROTECT_ROOT_FILE_RELATIVE_PATH === $plugin ) {
+	if (
```

Comment:
nitpick, I think we could invert the order of the checks, as `JETPACK_PROTECT_ROOT_FILE_RELATIVE_PATH === $plugin` is a much cheaper check.

Reply:
Good call, thanks!